db/migrate/20160129082610_add_object_changes_to_versions.rb

```diff
@@ -0,0 +1,10 @@
+class AddObjectChangesToVersions < ActiveRecord::Migration
+
+  # The largest text column available in all supported RDBMS.
+  # See `create_versions.rb` for details.
+  TEXT_BYTES = 1_073_741_823
+
+  def change
+    add_column :versions, :object_changes, :text, limit: TEXT_BYTES
```

Comment:
I thinks versions should be item_history, I think of the item changing it's self when you say version like color change